src/python/foglamp/model/config.py

```diff
@@ -0,0 +1,56 @@
+"""
+Environment variables
+
+# FOGLAMP_DB_PASSWORD
+"""
+import os
+import yaml
+import foglamp.config as config
+
+# TODO: write tests
+
+
+db_connection_string = None
+"""
+See http://docs.sqlalchemy.org/en/latest/core/engines.html
+"""
+
+
+def read_config():
+    """
+    reading the YAML config_file as defined via FOGLAMP_CONFIG_PATH
```

Comment:
should be: 
`"""reads database info from loaded configuration"""`
`""" if set using FOGLAMP_DB_PASSWORD environment variable, it will be overridden"""`